docs/assets/ovs-pipeline.svg

```diff
@@ -9,4248 +9,1821 @@
-         style="font-style:normal;font-variant:normal;font-weight:500;font-stretch:normal;font-size:4.93889px;font-family:Oswald;-inkscape-font-specification:'Oswald, Medium';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:center;writing-mode:lr-tb;text-anchor:middle;fill:#ffffff;fill-opacity:1;stroke-width:0.264583"
-         id="path1545" />
-      <path
-         d="m 170.9702,121.44521 v -4.0005 h 0.52846 v 4.0005 z"
-         style="font-style:normal;font-variant:normal;font-weight:500;font-stretch:normal;font-size:4.93889px;font-family:Oswald;-inkscape-font-specification:'Oswald, Medium';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:center;writing-mode:lr-tb;text-anchor:middle;fill:#ffffff;fill-opacity:1;stroke-width:0.264583"
-         id="path1547" />
-      <path
-         d="m 172.83964,121.48472 q -0.33584,0 -0.51858,-0.12841 -0.18274,-0.13335 -0.25682,-0.38523 -0.0692,-0.25682 -0.0692,-0.62724 v -0.65193 q 0,-0.3803 0.0741,-0.63218 0.0741,-0.25188 0.25682,-0.3803 0.18768,-0.12841 0.51364,-0.12841 0.36054,0 0.52846,0.14817 0.17287,0.14817 0.22719,0.42968 0.0543,0.27658 0.0543,0.66675 v 0.23213 h -1.12607 v 0.57291 q 0,0.17286 0.0296,0.28152 0.0346,0.10866 0.10371,0.16298 0.0741,0.0494 0.18768,0.0494 0.11854,0 0.1778,-0.0543 0.0642,-0.0593 0.084,-0.16298 0.0247,-0.10865 0.0247,-0.25188 v -0.17286 h 0.51364 v 0.10865 q 0,0.42969 -0.18274,0.67663 -0.18274,0.24694 -0.6223,0.24694 z m -0.31609,-1.74342 h 0.60749 v -0.25683 q 0,-0.1778 -0.0198,-0.29633 -0.0197,-0.12347 -0.084,-0.18274 -0.0593,-0.0642 -0.19262,-0.0642 -0.12841,0 -0.19755,0.0593 -0.0642,0.0593 -0.0889,0.2025 -0.0247,0.13829 -0.0247,0.38523 z"
-         style="font-style:normal;font-variant:normal;font-weight:500;font-stretch:normal;font-size:4.93889px;font-family:Oswald;-inkscape-font-specification:'Oswald, Medium';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:center;writing-mode:lr-tb;text-anchor:middle;fill:#ffffff;fill-opacity:1;stroke-width:0.264583"
-         id="path1549" />
-      <path
-         d="m 153.34781,127.61883 0.2025,-1.43228 h -0.21237 v -0.39511 h 0.27163 l 0.0692,-0.47908 h -0.33091 v -0.40005 h 0.38524 l 0.18767,-1.29399 h 0.52847 l -0.18768,1.29399 h 0.4445 l 0.18768,-1.29399 h 0.52352 l -0.18768,1.29399 h 0.20249 v 0.40005 h -0.25682 l -0.0741,0.47908 h 0.31609 v 0.39511 h -0.37042 l -0.20743,1.43228 h -0.52846 l 0.20743,-1.43228 h -0.43956 l -0.2025,1.43228 z m 0.78529,-1.82739 h 0.4445 l 0.0691,-0.47908 h -0.43956 z"
-         style="font-style:normal;font-variant:normal;font-weight:500;font-stretch:normal;font-size:4.93889px;font-family:Oswald;-inkscape-font-specification:'Oswald, Medium';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:nor
```

Comment:
Why there is an arrow from table 71 to 80?

Reply:
My fault, updated